README.md

```diff
@@ -1,2 +1,27 @@
 # click-apps-for-openwrt
-OpenWrt based click applications
+
+###Overview
+
+Applications to show how mikro-e [sensor clicks](http://www.mikroe.com/add-on-boards/sensors/) can be used with Ci40.
+
+###Application to detect motion using [motion click](http://www.mikroe.com/click/motion/)
+
+After starting the app move your hand above the motion click.
+
+Press ctrl+c to exit the application.
+
+Two options can be given:
+
+1) mikroBUS number to specify where motion click is connected.
+
+2) timeout to specify time between successive attempts.
+
+Default values for [mikroBUS](http://www.mikroe.com/mikrobus/) is 1 and for timeout is 5 seconds
```

Comment:
You are not accepting the timeout argument, then change the readme accordingly..